app/helpers/api/plans_helper.rb

```diff
@@ -12,8 +12,7 @@ def default_application_plan_data(service, plans)
   end
 
   def application_plans_data(plans)
-    plans.not_custom
-         .alphabetically
+    plans.alphabetically
```

Comment:
Why the change here?

Reply:
The data helper really does not care if "plans" are custom or not, it simply arranges the data for the UI.